public/content/dao/index.md

```diff
@@ -92,7 +92,7 @@ In 1977, Wyoming invented the LLC, which protects entrepreneurs and limits their
 
 ### A famous example {#law-example}
 
-[CityDAO](https://citydao.io) – CityDAO used Wyoming's DAO law to buy 40 acres of land near Yellowstone National Park.
+[CityDAO](https://www.citydao.io) – CityDAO used Wyoming's DAO law to buy 40 acres of land near Yellowstone National Park.
```

Comment:
Neither of these are working for me... not sure I'd fall back to archive for this one; would either find another example or just remove this entire section (remove `### A famous example` with this link)